src/components/menu/index.js

```diff
@@ -12,29 +15,47 @@ export const Menu = ({ switchTab }) => {
   const handleClick = () => {
     setIsOpen(!isOpen);
   };
+  const [isLogedIn, setIsLogedIn] = useState(false);
+
+  useEffect(() => {
+    const auth = localStorage.getItem('user');
+    if (auth) {
+      setIsLogedIn(true);
+    }
+  }, []);
   return (
     <div className="hamburger-menu">
       <input className="menuToggle" type="checkbox" />
       <button className="menu__btn" htmlFor="menuToggle" onClick={handleClick}>
         <img src={icon} alt="Menu button" />
       </button>
-      <ul className={`menu__box ${isOpen ? 'open' : 'close'}`}>
-        <li>
-          <button className="menu__item" onClick={() => changeTab('EDIT_PROFILE')}>
-            Edit profile
-          </button>
-        </li>
-        <li>
-          <button className="menu__item" onClick={() => changeTab('CREATE_TARGET')}>
-            Create target
-          </button>
-        </li>
-        <li>
-          <button className="menu__item" onClick={() => handleLogout()}>
-            Log Out
-          </button>
-        </li>
-      </ul>
+      {isLogedIn ? (
```

Comment:
You could move this line inside the `<ul>` to avoid repeating the logic to add the open/close class